docs/content/developer/iota-101/transactions/sponsored-transactions/use-sponsored-transactions.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 description: A guide to using sponsored transactions on the IOTA blockchain, including .
-tags: [ move, transaction, address]
+tags: [ move, transaction, address, rust-sdk]
```

Comment:
rust-sdk isn't used anywhere in this file? Just some Rust types, but they aren't from the rust-sdk

Reply:
```suggestion
tags: [ move, transaction, address]
```

Reply:
Thanks!